project_app/components/Header.js

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import AppBar from "@mui/material/AppBar";
+import Image from "next/image";
+import styled from "styled-components";
+import Button from "@mui/material/Button";
+import Chip from "@mui/material/Chip";
+import Stack from "@mui/material/Stack";
+import Card from "@mui/material/Card";
+
+function Header() {
+  return (
+    <>
+      <AppBar
+        style={{ height: "412px", backgroundColor: "#353535", width: "1440px" }}
+      >
+        <Container maxWidth="xl">
+          <h1 className="u-text--center">AIN'T BOARD</h1>
```

Comment:
where is this class been used in CSS `u-text--center`?

Reply:
Removed that